src/components/galleryscreen.js

```diff
@@ -101,10 +171,11 @@ class GalleryScreen extends Component {
                     </View>
                     
                     <View style={styles.bottomContainer}>
-                        <Icon style={styles.plusIcon} name='plus' size={60} type='evilicon' color='#686868'
-                        onPress={()=>console.log('Add album')}></Icon>
+                        <Icon style={styles.plusIcon} name='plus' size='60' type='evilicon' color='#686868'
```

Comment:
This should not be a string '60', leads to this error: Warning: Failed prop type: Invalid prop `fontSize` of type `string` supplied to `Text`, expected `number`.

I fixed it on the master branch so you can just change '60' back to {60}

Reply:
what chris said ^^